test/test_experiment_fragment.py

```diff
@@ -87,6 +146,71 @@ def build_fragment(self):
             self.create(InvalidBindFragment, [])
 
 
+class StrOptions(Enum):
+    first = "A"
+    second = "B"
+    third = "C"
+
+
+class IntOptions(Enum):
+    first = 1
+    second = 2
+    third = 3
+
+
+class EnumFragString(ExpFragment):
+    def build_fragment(self):
+        self.setattr_param("value_str",
+                           EnumParam,
+                           description="Enum string",
+                           default=StrOptions.first)
+
+    def run_once(self):
+        print(self.value_str.get())
```

Comment:
Doesn't this just add line noise when running the tests? To check whether the rebinding works, we could e.g. just set a class member to the `get()` result.